drivers/sensor/wsen_hids/hids.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT we_wsen_hids
+
+#include <string.h>
+
+#include <zephyr/sys/__assert.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/logging/log.h>
+
+#include "hids.h"
+
+LOG_MODULE_REGISTER(HIDS, CONFIG_SENSOR_LOG_LEVEL);
+
+struct str2odr {
+	const char *str;
+	HIDS_outputDataRate_t odr;
+};
+
+/* Maps ODR configuration string (CONFIG_HIDS_ODR) to enum value used by HIDS driver. */
+static const struct str2odr hids_odr_string_list[] = {
+	{ "1", HIDS_odr1Hz },
```

Comment:
please use designated initializer wherever possible, I think you dont need this if you used enum in the bindings

Reply:
True, has been removed.